how-to/add-mods/curseforge-app.md

```diff
@@ -0,0 +1,31 @@
+---
+hidden: true
+---
+
+# CurseForge App
+
+{% tabs %}
+{% tab title="Through the CurseForge App" %}
+1. Open the **CurseForge App**
+2. Select **Minecraft** from the grid or from the sidebar
+3. In **My Modpacks**, click on **Fabulously Optimized**
+4. Click on the **three dots** button
+5. Select **Profile Options**
+6. Check the **Allow content management for this profile** checkbox
+7. Click on **Done**
+8. Click on the **Puzzle** button
+9. Find the mod or mods you want to add and select them
+10. Once installed, launch the instance. Minecraft should open up
+11. If you can see the newly installed content in the Mod Menu, you're done!
+{% endtab %}
+
+{% tab title="Through the CurseForge website" %}
+1. Find the mod or mods you want to add from the [list of Fabric mods on CurseForge](https://www.curseforge.com/minecraft/search?gameVersionTypeId=4)
+2. Click on **Install** next to the latest version compatible with Fabric and Minecraft. The CurseForge App should open
```

Comment:
The website's global install button is actually pretty stupid so we should instruct users to use the Files tab:

```suggestion
2. Click on the mod details page, then **Files**
3. Find a file that is compatible with Fabric and the Minecraft version you're using, click on it
4. Click on **T̲ Install**. The CurseForge App should open
```
(of course fix the proceeding numbering too)